recommenders/evaluation/spark_evaluation.py

```diff
@@ -554,6 +556,7 @@ def __init__(
         self.df_user_diversity = None
         self.avg_diversity = None
         self.item_feature_df = item_feature_df
+        self.item_sim_measure = item_sim_measure
```

Comment:
Also please add the new argument in the docstring too.

Reply:
added into docstring.